.github/workflows/kb_sdk-actions.yml

```diff
@@ -0,0 +1,136 @@
+name: Basic kb-sdk tests
+
+on:
+  [push, pull_request]
+
+jobs:
+
+  test_kb-sdk_builds:
+    if: "!contains(github.event.head_commit.message, 'skip ci')"
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
```

Comment:
I suspect many of our machines are still on 12.04 or 14.04... @kkellerlbl?

Reply:
Unfortunately those systems are too ancient to be testable via GitHub actions!